bridge_adaptivity/api/backends/base_api_client.py

```diff
@@ -0,0 +1,44 @@
+import slumber
+
+
+class BaseApiClient(slumber.API):
+    """
+    Provide base class for the content source API.
+    """
+
+    def __init__(self, content_source):
+        self.content_source = content_source
+        super().__init__(self.url)
+
+    @property
+    def url(self):
+        return self.content_source.host_url
```

Comment:
lets have default param for api something like <host>/api/v1/

Reply:
done